ETTrace/ETTraceRunner/RunnerHelper.swift

```diff
@@ -34,12 +34,29 @@ class RunnerHelper {
     }
     
     func start() async throws {
-        print("Please open the app on the \(useSimulator ? "simulator" : "device")")
-        if !useSimulator {
-            print("Re-run with `--simulator` to connect to the simulator.")
+        if !useSimulator || !isPortInUse(port: Int(PTPortNumber)) {
+          // App is not ready to connect
+          if useSimulator {
+            let running = listRunningProcesses()
+            if !running.isEmpty {
+              print("\(running.count) app\(running.count == 1 ? " was" : "s were") found but \(running.count == 1 ? "it is" : "they are") not running ettrace.")
```

Comment:
Would be easier to follow if you did `running.count == 1 ? "1 app was found but it is not running" : "\(running.count) apps were found but they are not running"`